launchdarkly-server-sdk/src/client.rs

```diff
@@ -965,6 +962,137 @@ mod tests {
         ));
     }
 
+    #[test]
```

Comment:
We didn't have any all_flags_detail tests (that I could find), so added some basic ones here.